cmd/scollector/collectors/network_windows.go

```diff
@@ -314,6 +333,13 @@ func c_network_windows_tcp() (opentsdb.MultiDataPoint, error) {
 		Add(&md, "win.net.tcp.segments", v.SegmentsReceivedPersec, opentsdb.TagSet{"type": "received"}, metadata.Counter, metadata.PerSecond, descWinNetTCPv4SegmentsReceivedPersec)
 		Add(&md, "win.net.tcp.segments", v.SegmentsRetransmittedPersec, opentsdb.TagSet{"type": "retransmitted"}, metadata.Counter, metadata.PerSecond, descWinNetTCPv4SegmentsRetransmittedPersec)
 		Add(&md, "win.net.tcp.segments", v.SegmentsSentPersec, opentsdb.TagSet{"type": "sent"}, metadata.Counter, metadata.PerSecond, descWinNetTCPv4SegmentsSentPersec)
+		if (v.SegmentsPersec-winNetTCPSegmentsLastCount) != 0 &&
+			(v.SegmentsRetransmittedPersec > winNetTCPSegmentsLastRetransmitCount) &&
+			(v.SegmentsPersec > winNetTCPSegmentsLastCount) {
+			Add(&md, "win.net.tcp.retransmit_pct", float64(v.SegmentsRetransmittedPersec-winNetTCPSegmentsLastRetransmitCount)/float64(v.SegmentsPersec-winNetTCPSegmentsLastCount)*100, nil, metadata.Gauge, metadata.Pct, descWinNetTCPv4SegmentsRetransmit)
+		}
+		winNetTCPSegmentsLastRetransmitCount = v.SegmentsRetransmittedPersec
```

Comment:
Need to turn a counter into a gauge. If I just use the raw values, I get the percent for all time, which on sql is misleading (the retransmit doesn't reset the counter as often as the others)